src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -169,8 +139,42 @@ export default class SimidMapCreative extends BaseSimidCreative {
     const adContainer = document.getElementById('adContainer');
     adContainer.appendChild(returnToAdButton);
     adContainer.appendChild(skipAdButton);
+    this.createTravelDisplay_();
+    this.simidMap_.setTravelMethodElement(document.getElementById("travel_method"));
+    this.simidMap_.setTimeDisplayElement(document.getElementById("time_display"));
+  }
 
-    this.displayMap_(this.userCoordinates_);
+    /**
+     * Creates an option element representing a mode of travel.
+     * @param {!string} travelMode A string representing the
+     *   given mode of travel.
+     * @private 
+     */
+    createTravelOption_(travelMode) {
```

Comment:
Could it be ideal to have all travel option stuff in the class that deals with maps?  I suppose it could make sense to keep it here too though.

It must be here, but where is the click handler for when the user changes travel options.

Reply:
decided to leave in creative class since it doesn't directly involve the maps api